sdcm/provision/scylla_yaml/scylla_yaml.py

```diff
@@ -339,6 +339,8 @@ def set_authorizer(cls, authorizer: str):
     audit_categories: str = None  # None
     audit_tables: str = None  # None
     audit_keyspaces: str = None  # None
+    force_gossip_topology_changes: bool = None  # False
+    enable_tablets: bool = None  # False
```

Comment:
bit misleading comment - 6.0.0 comes with this set explicitly to `True` - I think worth to add this info (like we do for e.g. rpc_port).

Reply:
it is set in scylla.yaml. but default fault is False. i added appropriate comment. if it set true by default, then for old version  this parameter will be set true automatically and could fail startup